src/helpers/utils.js

```diff
@@ -0,0 +1,22 @@
+const includes = (container, value) => {
```

Comment:
Awesome stuff. We might want to refactor some small part of the app to use these 🥇